t/issue/948.t

```diff
@@ -26,7 +26,7 @@ sub Rex::Commands::File::file {
   my ( $name, %params ) = @_;
 
   my $fmt = get_image_format();
-  like $params{content}, qr|<driver name="qemu" type="$fmt"/>|,
+  like $params{content}, qr|<driver name="qemu" type="$fmt"\s*/>|,
```

Comment:
Currently we don't really have better tests, so let's say we accept the little extra risk on top, and choose to trust your experience in production with the newly added features.

ps.: while that would definitely be another scope and story, any better test coverage for this part of the code base would be warmly welcome :)

Reply:
I'll be happy to follow up with more tests in the future when I have the time. Noted.